platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/HappyPathTest.java

```diff
@@ -35,5 +36,11 @@ void testHappyPath(TestEnvironment env) throws InterruptedException {
                         LogFilter.ignoreMarkers(LogMarker.STARTUP),
                         LogFilter.ignoreNodes(network.getNodes().getFirst()))
                 .validateRemaining(Profile.DEFAULT);
+
+        // Validations
+        env.validator().validateRemaining(Profile.DEFAULT);
```

Comment:
Merge error? `validateRemaining()` is already called before.

Reply:
Correct, removed.